stubs/requests/requests/adapters.pyi

```diff
@@ -64,7 +81,10 @@ class HTTPAdapter(BaseAdapter):
     def init_poolmanager(self, connections, maxsize, block=False, **pool_kwargs): ...
     def proxy_manager_for(self, proxy, **proxy_kwargs): ...
     def cert_verify(self, conn, url, verify, cert): ...
-    def build_response(self, req, resp): ...
+    def build_response(self, req: PreparedRequest, resp: urllib3.BaseHTTPResponse) -> Response: ...
+    def build_connection_pool_key_attributes(
+        self, request: PreparedRequest, verify: bool | str, cert: str | tuple[str, str] | None = None
```

Comment:
According to the implementation, `verify` can also be `None`, but the docstring makes me think that is not desired for this method.